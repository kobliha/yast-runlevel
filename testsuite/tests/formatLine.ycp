// tests formatLine
{
    include "testsuite.ycp";
    include "runlevel/ui.ycp";

    string s = "";
    list splited = [];

    s = formatLine (["a", ], 15);
    splited = splitstring (s, "\n");
    DUMP (splited);

    s = formatLine (["a", "b", "c", "d", "e", ], 15);
    splited = splitstring (s, "\n");
    DUMP (splited);

    s = formatLine (["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", ], 15);
    splited = splitstring (s, "\n");
    DUMP (splited);
}
