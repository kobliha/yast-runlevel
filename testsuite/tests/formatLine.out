Dump	["     a"]
Dump	["     a, b, c, d, e"]
Dump	["     a, b, c, d, e,", "     f, g, h, i, j,", "     k, l"]
